interface/wx/notebook.h

```diff
@@ -191,6 +191,19 @@ class wxNotebook : public wxBookCtrlBase
     */
     virtual void SetPadding(const wxSize& padding);
 
+    /**
+        Returns wxTOP / wxBOTTOM / wxRIGHT / wxLEFT
```

Comment:
I'd like to expand the description a bit and we also should indicate that it's only available in 3.3 and later:

```suggestion
        Returns one of wxTOP, wxBOTTOM, wxRIGHT or wxLEFT.
        
        This is a convenience function mapping wxBK_TOP etc styles to one of the
        wxDirection enum elements.
        
        @since 3.3.0
```